galaxy.yml

```diff
@@ -25,20 +49,32 @@ tags:
   - ims
   - zos_ims
   - z_and_cloud_modernization_stack_operator
+  - zoscb
+
+# Collections that this collection requires to be installed for it to be usable. The key of the dict is the
+# collection label 'namespace.name'. The value is a version range
+# L(specifiers,https://python-semanticversion.readthedocs.io/en/latest/#requirement-specification). Multiple version
```

Comment:
I haven't added a link to yml comment in a while. Is L(string, url) the correct syntax for this?